t/tasks/inventory/macos/storages.t

```diff
@@ -125,7 +126,7 @@ my %testsSerialATA = (
             SERIAL       => 'WD-WCARY1264478',
             MODEL        => 'WDC WD2500AAJS-40VWA1',
             FIRMWARE     => '58.01D02',
-            DISKSIZE     => FusionInventory::Agent::Task::Inventory::MacOS::Storages::_fromBytesToMegaBytes(250059350016),
```

Comment:
Why computing value to test ? I think tested values should be defined statically otherwise you may miss bug introduced in used API